.github/workflows/build-yocto.yml

```diff
@@ -14,10 +14,19 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Run kas lock
+        run: |
+          kas dump --update --lock --inplace ci/base.yml
+
       - name: Run yocto-check-layer
         run: |
           ci/yocto-check-layer.sh
 
+      - uses: actions/upload-artifact@v4
+        with:
+          name: kas-lock
+          path: ci
```

Comment:
you really only need the base.lock file, not the whole folder.

Reply:
I create a new folder to store the lock files